.github/dependabot.yml

```diff
@@ -1,17 +1,17 @@
 version: 2
 updates:
-  - package-ecosystem: "pip"
+  - package-ecosystem: "github-actions"
```

Comment:
Is there a reason behind the order change ?

Reply:
https://github.com/StevenBlack/hosts/pull/2623#issuecomment-2034699665

I just keep actions first and then any package managers